### PR TITLE
Fix navigator sidebar to show help icon and chevron in mobile view

### LINKED ISSUE #3862: [UI] Help and Chevron not visible in mobile view
#### Current Behavior
The Help button and the Chevron are not visible in the Meshery's dasboard in the mobile view.

#### Desired Behavior
The Help button and the Chevron should be visible in the Meshery's dasboard in the mobile view too.

#### Screenshots/Logs
Screenshot 2021-08-12 at 9 37 22 PM

#### Environment

- Browser: Chrome Safari 
- Host OS: Mac 
- Meshery Server Version: v0.5.45
- Meshery Client Version: v0.5.45
- Platform: Docker 

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Yash Agarwal <yagarwal1307@gmail.com>

This PR fixes #3862

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
Checked on local, LGTM 🎉, Thanks @yagarwal1307 , that was super fast 😅.

File: ui/components/Navigator.js [hunk 1]
```diff
@@ -147,13 +147,11 @@ const styles = (theme) => ({
...
-  fixedSidebarFooter : { display : "none",
-    [theme.breakpoints.up("sm")] : {
-      display : "flex",
-      flexDirection : "column",
-      "margin-top" : "auto",
-      "margin-bottom" : "0.5rem",
-    }, },
+  fixedSidebarFooter : { display : "flex",
+    flexDirection : "column",
+    "margin-top" : "auto",
```
Comment (sudo-NithishKarthik) [inline ui/components/Navigator.js:152]: lets not mix up the convention, we are using camel case everywhere for declaring style properties, let's just use the same

Comment (yagarwal1307) [reply ui/components/Navigator.js:152]: I have just removed some lines, didn't add any! So there is no change in convention because of my changes!

Comment (sudo-NithishKarthik) [reply ui/components/Navigator.js:152]: Ok, no issues @yagarwal1307 ✌🏽  , can you change them to camel casing ?

**Review by warunicorn19 — APPROVED** [review]
@sudo-NithishKarthik, this also magically fixed the UI test fail🤔.

**Comment by sudo-NithishKarthik** [comment]
> @sudo-NithishKarthik, this also magically fixed the UI test fail🤔.

No, that's because I ran the job again. 😄

**Review by sudo-NithishKarthik — APPROVED** [review]
LGTM
Thank you @yagarwal1307 🎉

**Comment by warunicorn19** [comment]
> > @sudo-NithishKarthik, this also magically fixed the UI test fail🤔.
> 
> No, that's because I ran the job again. 😄

😂😂😂Ohh okay

**Pull request merged**